pkg/descheduler/descheduler_test.go

```diff
@@ -44,15 +44,20 @@ func TestTaintsUpdated(t *testing.T) {
 	}
 	rs.Client = client
 	rs.DeschedulingInterval = 100 * time.Millisecond
+	errChan := make(chan error, 1)
+	defer close(errChan)
 	go func() {
 		err := RunDeschedulerStrategies(ctx, rs, dp, "v1beta1", stopChannel)
+		errChan <- err
+	}()
+	select {
+	case err := <-errChan:
 		if err != nil {
 			t.Fatalf("Unable to run descheduler strategies: %v", err)
 		}
-	}()
+	case <-time.After(300 * time.Millisecond):
+	}
 
-	// Wait for few cycles and then verify the only pod still exists
```

Comment:
I think this comment is useful, can you keep it?

Reply:
Done.
Thanks for review ^-^